README.md

```diff
@@ -16,59 +18,67 @@ transformers:
   - sass
 ```
 
-After adding the transformer your `.sass` and `.scss` files will be automatically transformed to
+After adding the transformer all your `.sass` and `.scss` files will be automatically transformed to
```

Comment:
I think there should be a `,` after "After adding the transformer". I started reading this sentence a few times until I got what it means ;-)